polkadot/runtime/rococo/src/lib.rs

```diff
@@ -150,7 +150,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("rococo"),
 	impl_name: create_runtime_str!("parity-rococo-v2.0"),
 	authoring_version: 0,
-	spec_version: 1_006_002,
+	spec_version: 1_006_003,
```

Comment:
@EgorPopelyaev is this ok? or change to 1_700_000?  1.7.0 release was branched off yesterday, and if we need  to release BridgeHubRococo from master

Reply:
1.7 would be on the release branch then, not master.